src/api-user/index.js

```diff
@@ -17,29 +18,42 @@ const axios = require("axios").default;
       githubUsername: author.login,
     };
 
-    
-    if (currentLabel === 'EddieHub-API-add') {
+    if (labels.includes(currentLabel)) {
       try {
         console.log(body);
-        await axios.post(apiURL, body, {
-          headers: { ...authHeader },
-        });
-        
-        await client.issues.createComment({
-          owner: payload.repository.owner.login,
-          repo: payload.repository.name,
-          issue_number: payload.issue.number,
-          body: 'You have been ADDED to the EddieHub API',
-        });
+        if (currentLabel === "EddieHub-API-add") {
+          await axios.post(apiURL, body, {
+            headers: { ...authHeader },
+          });
+        } else {
+          await axios.delete(`${apiURL}/${author}`, {
+            headers: { ...authHeader },
+          });
+        }
+        if (currentLabel === "EddieHub-API-add") {
+          await client.issues.createComment({
+            owner: payload.repository.owner.login,
+            repo: payload.repository.name,
+            issue_number: payload.issue.number,
+            body: "You have been ADDED to the EddieHub API",
+          });
+        } else {
+          await client.issues.createComment({
+            owner: payload.repository.owner.login,
+            repo: payload.repository.name,
+            issue_number: payload.issue.number,
+            body: "You have been REMOVED from the EddieHub API",
+          });
+        }
```

Comment:
Looks great 👍 I have 1 suggestions, would it be easier better for readability to have 2 different `if` statements rather than both the same?

For example...

```js
        if (currentLabel === "EddieHub-API-add") {
           await axios.post(apiURL, body, {
             headers: { ...authHeader },
           });

           await client.issues.createComment({
             owner: payload.repository.owner.login,
             repo: payload.repository.name,
             issue_number: payload.issue.number,
             body: "You have been ADDED to the EddieHub API",
           });
         }

         if (currentLabel === "EddieHub-API-delete") {
           await axios.delete(`${apiURL}/${author}`, {
             headers: { ...authHeader },
           });

           await client.issues.createComment({
             owner: payload.repository.owner.login,
             repo: payload.repository.name,
             issue_number: payload.issue.number,
             body: "You have been REMOVED from the EddieHub API",
           });
         }
```